easybuild/easyconfigs/m/MultiQC/MultiQC-1.10.1-foss-2020b.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'PythonBundle'
```

Comment:
@genio0815 same here as in #14599: assuming you based this easyconfig on the existing MultiQC easyconfigs, could you please reinstate the comments at the top that all the previous MultiQC versions have (and feel free to add a line that says you updated it for version 1.10.1)? So, something like:
```suggestion
# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
# Adam Huffman
# The Francis Crick Institute
# Elements derived from work by Pablo Escobar
# sciCORE - University of Basel
# SIB Swiss Institute of Bioinformatics 

# Note that Click in Python 3 requires that you change your locale to unicode before invoking your Python script.
# See: https://click.palletsprojects.com/en/7.x/python3/

easyblock = 'PythonBundle'
```